test/test-songs.js

```diff
@@ -0,0 +1,95 @@
+process.env.NODE_ENV = 'test';
+
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+const sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const server = require('../index');
+let should = chai.should();
+const spotifyWebApi = require('spotify-web-api-node');
+
+const songController = require('../controllers/songController');
+const Stub = require('./stub');
+const mocks = require('./mocks');
+
+chai.use(chaiHttp);
+chai.use(sinonChai);
+
+let stubGetUserPlaylists, stubGetPlaylistTracks, stubLikeSongs;
+
+describe('Songs:', () => {
+  let request, response;
+  beforeEach(() => {
+    request = {
+      spotifyApi: new spotifyWebApi(mocks.spotifyObj)
+    };
+
+    stubGetUserPlaylists = Stub.createStub(request.spotifyApi, 'getUserPlaylists', mocks.playlists);
```

Comment:
All this stub thing is repeated on every test file. This should be refactored in the future.